modules/installing-ocp-agent.adoc

```diff
@@ -0,0 +1,159 @@
+// Module included in the following assemblies:
+//
+// * installing-with-agent/installing-with-agent.adoc
+
+:_content-type: PROCEDURE
+[id="installing-ocp-agent_{context}"]
+= Installing {product-title} with the Agent-based Installer
+
+The following procedure deploys a single-node {product-title} and you can use this procedure as a basis and modify according to your requirements.
+
+.Procedure
+
+. Navigate to the link:https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/latest-{product-version}/[OpenShift mirror site] and download the latest version of the tarball that matches your operating system.
+
+. Extract the binary.
++
+[source,terminal]
+----
+$ tar xvf <file> -C ~/.local/bin
+----
++
+The file contains the Agent-based installer.
+
+. Install `nmstate` dependency.
++
+[source,terminal]
+----
+$ sudo dnf install /usr/bin/nmstatectl -y
+----
+
+. Place the `openshift-install` binary in a directory that is on your PATH.
+
+. Create a directory to store the cluster manifests.
++
+[source,terminal]
+----
+$ mkdir ~/<directory_name>
+----
+
+. Create the `install-config.yaml` file.
++
+[source,yaml]
+----
+cat << EOF > ./cluster-manifests/install-config.yaml
+apiVersion: v1
+baseDomain: test.example.com
+compute:
+- architecture: amd64
+  hyperthreading: Enabled
+  name: worker
+  replicas: 0
+controlPlane:
+  architecture: amd64
+  hyperthreading: Enabled
+  name: master
+  replicas: 1
+metadata:
+  name: sno-cluster <1>
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  machineNetwork:
+  - cidr: 192.168.111.0/16
+  networkType: OpenShiftSDN
+  serviceNetwork:
+  - 172.30.0.0/16
+platform:
+  none: {}
+pullSecret: '<pull_secret>' <2>
+sshKey: |
+  '<ssh_pub_key>' <3>
+  EOF
+----
++
+<1> Required.
+<2> Enter your pull secret.
+<3> Enter your ssh public key.
+
+. Create the `agent-config.yaml` file.
++
+[source,yaml]
+----
+  cat > agent-config.yaml << EOF
+  kind: AgentConfig
+  metadata:
+    name: sno-cluster
+  rendezvousIP: 192.168.111.80 <1>
+  hosts:
```

Comment:
We should call out that is is a SNO example. The number of hosts defined here should match the total number of hosts defined in the `install-config.yaml`. (The sum of compute.replicas and controlPlane.replicas)

In other words: 
- If you have 3 masters and 2 workers in install-config you need 5 hosts defined here.
- If you have 3 masters and 0 workers in install-config you need 3 hosts defined here.
- If you have 1 masters and 0 workers in install-config you need 1 host defined here.

I'm not sure how to word this in the documentation though.